logstash_async/formatter.py

```diff
@@ -21,10 +21,40 @@
     import simplejson as json
 
 
+class _LogstashMessageSchema:
```

Comment:
Only cosmetic: we could move this into the `LogstashFormatter` class as it is done with the others.

Reply:
Done.